integration/flows/popover.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+
+test('popover closes .popover-content when an outer element is clicked', async ({ page }) => {
+	// Go to http://localhost:1111/components/popover.html
+	await page.goto('http://localhost:1111/components/popover.html');
```

Comment:
nit- relative url to conform with:

>It is also recommended to specify [testOptions.baseURL](https://playwright.dev/docs/api/class-testoptions#test-options-base-url) in the config, so that tests could use relative urls.
https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests